.github/workflows/publish_pypi_coinbase_agentkit.yml

```diff
@@ -17,7 +17,10 @@ jobs:
       id-token: write
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+      - run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
See here: https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-using-the-built-in-token